satpy/etc/composites/fci.yaml

```diff
@@ -355,18 +355,18 @@ composites:
 
   cloud_phase:
     description: >
-      Equal to cloud_phase recipe, but with additional sunz_reducer modifier to avoid saturation at the terminator.
+      Equal to cloud_phase recipe, but with the effective_solar_pathlength_corrected modifier to avoid saturation at the terminator.
     references:
       EUMETRAIN Quick Guide: https://resources.eumetrain.org/rgb_quick_guides/quick_guides/CloudPhaseRGB.pdf
       Recipe: https://resources.eumetrain.org/RGBguide/recipes/RGB_recipes.pdf
     compositor: !!python/name:satpy.composites.GenericCompositor
     prerequisites:
       - name: nir_16
-        modifiers: [sunz_corrected, sunz_reduced]
+        modifiers: [ effective_solar_pathlength_corrected ]
       - name: nir_22
-        modifiers: [sunz_corrected, sunz_reduced]
+        modifiers: [ effective_solar_pathlength_corrected ]
       - name: vis_06
-        modifiers: [sunz_corrected, rayleigh_corrected, sunz_reduced]
+        modifiers: [ effective_solar_pathlength_corrected, rayleigh_corrected ]
```

Comment:
So at the RGB workshop, in was said that Rayleigh correction for 0.6 µm was optional, except for true color where it was recommended IIRC.
Now I don't have any particular opinion myself on if it needs to be applied or not for cloud phase and cloud type, but it would be nice to be consistent unless there is a specific reason to have it or not.
Not applying the correction could also improve performance.

Reply:
Good point.  I prefer without, because I think it introduces an artefact.  Removed in abd4f69bb.